Makefile

```diff
@@ -9,6 +9,8 @@ YARN_CACHE_DIR=~/.yarncache
 ISOLATED_VM_VERSION_COMMAND="require('./node_modules/isolated-vm/package.json').version"
 ISOLATED_VM_VERSION=$(shell node -p -e $(ISOLATED_VM_VERSION_COMMAND))
 
+PIPENV := PYTHONPATH=${ROOTDIR} PIPENV_IGNORE_VIRTUALENVS=1 pipenv
```

Comment:
Is this making any assumptions about what system python is running? Or does the system-installed pipenv seem sufficient to produce a hermetic python 3.9 environment?

Context is that we have problems when relying on system python, it can differ a lot from machine to machine, plus on the eng team about half the team uses linux devboxes on AWS and the rest use macs. On the `coda` side we have to set up a standardized pipenv as part of the bootstrap process. I guess if the built-in pipenv is sufficient to bootstrap isn't a consistent python 3.9 environment then this will work.

Reply:
I guess we check for a valid python here: https://github.com/coda/coda/blob/main/setup-env.sh#L164 I think that implies that we still install it somewhere else in `make bs` though.

In my experience using python requires some flavor of setup-env.sh script to bootstrap one's environment, we'll see how far we can get without one but it kind of feels inevitable. Totally fine if we don't get all of it in one PR though, we can just do individual meaningful pieces as long as they don't break `make build`.

Reply:
Digging around, it seems like `make bs` just fails if the Python version isn't a match with the Pipfile:

https://github.com/coda/coda/blob/main/scripts/dev/check-python-version.sh

We can try to replicate that in this repo, but I agree that we probably add that in later as needed.

Reply:
This works on Mac but unfortunately this is what I'm getting when I run `make bs` on my linux devbox:

```
 admin@devbox-jonathan  ~/src/packs-sdk   ek-mkdocs  make bs
yarn config v1.22.5
success Set "cache-folder" to "/home/admin/.yarncache".
Done in 0.08s.
yarn install v1.22.5
[1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.
Done in 0.39s.
Warning: Python 3.9 was not found on your system…
You can specify specific versions of Python with:
  $ pipenv --python path/to/python
make[1]: *** [Makefile:28: _bootstrap-python] Error 1
make: *** [Makefile:33: bootstrap] Error 2
```

So can't proceed with this quite as-is, either need to make all of this a manual step for now to unblock you while we figure this out, or need to deal with our linux env not having python 3.9 available by default.